RAMP/packer.py

```diff
@@ -5,6 +5,8 @@
 import semantic_version
 import tempfile
 from subprocess import Popen, PIPE
+from __init__ import eprint
```

Comment:
Is it common to specify `__init__` as an import?

Reply:
I guess it's cleaner to add a common.py file.